src/components/Layout/Layout.tsx

```diff
@@ -1,14 +1,14 @@
 import styled, { css } from 'react-emotion';
 
-export const Content = styled('div')`
-  && {
-    background-color: #ffffff;
-    flex: 1;
-    display: flex;
-    position: relative;
-    flex-direction: column;
-  }
-`;
+export const Content = styled('div')({
+  '&&': {
+    backgroundColor: '#ffffff',
```

Comment:
For future tickets `#ffffff'` should be reusable.

Reply:
Added to #107